File: 92-Printing/external/xtensive/SampleUsbPrinting/app/src/main/java/com/example/moljac/sampleusbprinting/usbprinter/MainActivity.java
package com.example.moljac.sampleusbprinting;

import com.example.moljac.sampleusbprinting.PrintOrder;

import android.app.ActionBar;
import android.app.Activity;
import android.app.AlertDialog;
import android.content.Context;
import android.os.Bundle;
import android.util.Log;
import android.widget.Toast;

public class MainActivity extends Activity{
	//private ConfigAdapter datasource;
	
	protected void onCreate(Bundle icicle) {
		 super.onCreate(icicle);
	     setContentView(R.layout.activity_main);
	     //String msg = 'This is a test message';
	     PrintOrder printer = new PrintOrder();
	     printer.Print(this,"ddsasdsadasdasdsa");
	}
}

File: tutorial-samples/45-Toolbar/Toolbar-API-v4x-AppCompatSupport/SupportToolbar/MainActivity.cs
using System;

using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;
using Android.Support.V7.App;

namespace SupportToolbar
{
	[Activity( MainLauncher = true, Theme="@style/MyTheme")]
	public class MainActivity :
	 //AppCompatActivity
	 ActionBarActivity
	{

		protected override void OnCreate (Bundle bundle)
		{
			base.OnCreate (bundle);

			// Set our view from the "main" layout resource
			SetContentView (Resource.Layout.Main);

			var toolbar = FindViewById<Android.Support.V7.Widget.Toolbar> (Resource.Id.toolbar);

			SetSupportActionBar (toolbar);
		}


		public override bool OnCreateOptionsMenu (IMenu menu)
		{
			MenuInflater.Inflate (Resource.Menu.menu, menu);
			return base.OnCreateOptionsMenu (menu);
		}
		public override bool OnOptionsItemSelected (IMenuItem item)
		{	
			Toast.MakeText(this, "Top ActionBar pressed: " + item.TitleFormatted, ToastLength.Short).Show();
			return base.OnOptionsItemSelected (item);
		}

	}
}


